recipes/gcc/all/conanfile.py

```diff
@@ -1,12 +1,14 @@
 from conan import ConanFile
 from conan.tools.gnu import Autotools, AutotoolsToolchain
+# from conan.tools.gnu.get_gnu_triplet import _get_gnu_triplet
 from conan.errors import ConanInvalidConfiguration
 from conan.tools.layout import basic_layout
 from conan.tools.apple import XCRun
 from conan.tools.files import copy, get, replace_in_file, rmdir, rm
 from conan.tools.build import cross_building
 from conan.tools.env import VirtualBuildEnv
 from conan.tools.microsoft import is_msvc
+# from conan.tools.build.cross_building import get_cross_building_settings
```

Comment:
Please, cleanup

Reply:
I will - I've left this here for the first review by @jcar87 as it relates to https://github.com/conan-io/conan/issues/12789. Perhaps there's a way of packaging this library that means the gnu triplet isn't required, as it's presently hardcoded: https://github.com/conan-io/conan-center-index/pull/15128/files#diff-91c51d260a5e804807092c0510143447d89f4fc2d42ac14a2203c0d71cd6abb8R166